src/hooks/useBasisForm/validation/validateRadioGroup.ts

```diff
@@ -0,0 +1,18 @@
+import { RadioOptions } from "../../../components/internal/InternalRadioGroup";
+import { InternalRadioGroupProps } from "../../../components/RadioGroup/types";
+
+const isRadioGroupEmpty = (value: string, options: RadioOptions) =>
+  options.findIndex((option) => option.value === value) === -1;
+
+export const validateRadioGroup = (
+  value: string,
+  props: Record<string, any>
+) => {
+  const { options } = props as InternalRadioGroupProps;
```

Comment:
Can't type `props: InternalRadioGroupProps` in the args?

Reply:
see comment above